include/grpcpp/grpc_library.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2018 gRPC authors.
+ * Copyright 2022 gRPC authors.
```

Comment:
Please switch to newer style C++ header/copyright comments in this the file (e.g, src/core/lib/event_engine/iomgr_engine/ev_epoll1_linux.cc)

Reply:
Done